RELEASES.md

```diff
@@ -1,3 +1,69 @@
+Version 1.22.0 (2017-11-23)
+==========================
+
+Language
+--------
+- [`non_snake_case` lint now allows extern no-mangle functions][44966]
+- [Now accepts underscores in unicode escapes][43716]
+- [`#![feature(const_fn)]` is now no longer required for using
+  const functions][43017] Still required for creating constant functions.
+
+Compiler
+--------
+- [rustc now defaults to having 32 codegen units at debug][44853]
+- [rustc will no longer inline in codegen units when compiling for debug][45075]
+  This should decrease compile times for debug builds.
+- [strict memory alignment now enabled on ARMv6][45094]
+- [Remove support for the PNaCl target `le32-unknown-nacl`][45041]
+
+Libraries
+---------
+- [Allow atomic operations up to 32 bits
+  on `armv5te_unknown_linux_gnueabi`][44978]
+- [`Box<Error>` now impls `Cow<str>`][44466]
+- [`std::mem::Discriminant` is now guarenteed to be `Send + Sync` if `T`
+  is `Send + Sync`][45095]
```

Comment:
`Discriminant<T>` is now Send/Sync regardless of T. It's a bug fix, before it was never Send/Sync.